steps/jira-issue-status-update/0.1.2/step.yml

```diff
@@ -0,0 +1,58 @@
+title: Update JIRA Issue Status
+summary: Change status of JIRA issues.
+description: Automate process of moving issues from 'In Progress' to 'Code Review'
+  or 'In Test'.
+website: https://github.com/MiSikora/bitrise-step-update-jira-issue-status
```

Comment:
The website, source_code_url, and support_url URLs point to a non-existent repo.

Reply:
Thanks for the catch! Probably some leftover when I was deciding how to name repo and step.